README.md

```diff
@@ -1,8 +1,71 @@
-# Jarivs Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Mitchell Acton . Jarvis Consulting
 
+Graduated with an Master of Engineering degree in Mechanical and Materials Engineering from the University of Western Ontario, and a Bachelor of Applied Science degree in Nanotechnology Engineering from the University of Waterloo. I gained exposure and experience in data analytics and software development during school, during internships, through online courses, and through work experience after graduating. Eventually this became my main focus at work, so I decided to formally make the switch to a full time software development role with Jarvis. Since starting with Jarvis, I have become comfortable working in an Agile/Scrum development cycle, have gained valuable experience with Linux, PostgreSQL database management, and Java. I'm looking forward to getting the opportunity to put these tools to use along with my existing data analysis and machine learning skills to make high quality software products with significant business impacts.
```

Comment:
I graduated with a Master of Engineering degree from the University of Western Ontario and a Bachelor of Applied Science degree in Engineering from the University of Waterloo. ~I gained exposure and experience in data analytics and software development during school, during internships, through online courses, and through work experience after graduating. Eventually, this became my main focus at work, so I decided to formally make the switch to a full time software development role with Jarvis.~ Since starting with Jarvis, I have become comfortable working in an Agile/Scrum development cycle, have gained valuable experience with Linux, RDBMS database management, and Java. I'm looking forward to getting the opportunity to put these tools to use along with my existing data analysis and machine learning skills to make high quality software products with significant business impacts.

Update the crossed section. Please do not mention that you switched or learning stuff online. Instead, you shall mention your interest in the industry or skill sets. Please join the office hour if you have more questions.